core/shared/src/main/scala/coursier/util/Tree.scala

```diff
@@ -1,57 +1,74 @@
 package coursier.util
 
-import scala.collection.mutable
-import scala.collection.mutable.ArrayBuffer
+import scala.annotation.tailrec
 
-object Tree {
+private[util] object Tree {
+
+  def apply[T](roots: Seq[T])(children: T => Seq[T],
+                              show: T => String): String = {
```

Comment:
The migration manager is a very Interesting plugin. But it seems that this is a false positive. If I understand the message correct, that it complains that `roots` is now a `Seq[T]` and was a `IndexedSeq[T]` but since `IndexedSeq` extends `Seq` directly this should be compatible even for the JVM, but I'm no expert. I made all objects within the util package private. I think users should not use this objects.